<?xml version="1.0"?>
<launch>

  <!-- Configuring control package-->
  <node respawn="false" pkg="tcc" type="tcc" name="tcc" output="screen">
      <param name="frame_id" type="string" value="world" />
      <param name="sim_type" type="string" value="sim_st" />
      <param name="mpc_sim" type="bool" value="false" />
      <remap from="/vins_estimator/odometry" to="/NTU_internal/drone_feedback" />   
  </node>
</launch>